File: book-network-ui/src/models/BorrowedBookResponse.js
class BorrowedBookResponse {
  #id;
  #title;
  #authorName;
  #returned;
  #returnApproved;

  constructor(id, title, authorName, returned, returnApproved) {
    this.#id = id;
    this.#title = title;
    this.#authorName = authorName;
    this.#returned = returned;
    this.#returnApproved = returnApproved;
  }

  get id() {
    return this.#id;
  }

  get title() {
    return this.#title;
  }

  get authorName() {
    return this.#authorName;
  }

  get returned() {
    return this.#returned;
  }

  get returnApproved() {
    return this.#returnApproved;
  }
}

export default BorrowedBookResponse;

File: book-network/src/main/java/dev/aniketkadam/book/book/BorrowedBookResponse.java
package dev.aniketkadam.book.book;

import lombok.*;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class BorrowedBookResponse {
    private String id;
    private String title;
    private String authorName;
    private String isbn;
    private double rate;
    private boolean returned;
    private boolean returnApproved;
}
